crates/red_knot/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "red_knot"
-version = "0.0.0"
+version.workspace = true
```

Comment:
I don't think this works for us. Red Knot will use its own versioning schema

Reply:
It's maybe for 1-2 more months but we then plan on releasing red knot with its own versioning schema.